packages/pci-sdk-iframe/src/pages/app/App.tsx

```diff
@@ -79,6 +79,7 @@ export default function App() {
 						ctaText={labels.setPinSubmitButton}
 						handleSubmit={handlePinSubmit}
 						id="pin-input"
+						maxlength={4}
```

Comment:
Can we do this for the setNewPin form??

Reply:
AFAIK a card PIN always has 4 digits ( as an industry standard for ATM's? ). 
Good to double-check with BE though 💆🏽‍♂️

Reply:
Checked the doubt with BE. Waiting for a response 👍🏻

Reply:
Confirmed by BE! 🚀